src/components/Tool/FixPhoneNumbers/FixPhoneNumbers.tsx

```diff
@@ -124,53 +115,66 @@ const FixPhoneNumbers: React.FC<Props> = ({
 }: Props) => {
   const { classes } = useStyles();
   const { enqueueSnackbar } = useSnackbar();
-  const { appName } = useGetAppSettings();
+  const appName = process.env.APP_NAME ?? 'MPDX';
```

Comment:
I think `useGetAppSettings` is a better way of doing this. Was there a reason you switched?